### ISSUE TITLE
Add Django CMS toolbar to admin pages

### ISSUE THREAD

**Issue by leetucker** [leetucker/django-danceschool#30]
Right now, if you go to any Django admin page directly (e.g. /admin/), you lose the Django CMS toolbar.  This can be confusing, because many editing functions such as adding Series or Related Items are actually done directly in the Django admin.

In principle, Django CMS provides overlay functionality that makes this change unnecessary (see http://docs.django-cms.org/en/release-3.4.x/how_to/toolbar.html).  However, in the experience of Boston Lindy Hop we found this to be buggy, and so we changed all of the sideframe/overlay links to be direct links to admin pages.

Getting the toolbar on the admin pages should just involve adding the appropriate template tag to an overridden version of the admin base template.  However, there is the potential issue for repeated loading of toolbars if an admin page is loaded via an overlay or sideframe.  Since overlays and sideframes are just iframes, getting the Django CMS toolbar on admin pages should be achievable by just overriding the admin base template and adding a piece of Javascript that loads the toolbar only if the page is not in an iframe.

Also, this change will require the overridden base admin to be loaded *before* templates from django.contrib.admin, and so we will need an app, similar to the djangocms_admin_style app, which is listed in INSTALLED_APPS before django.contrib.admin.

**Comment by leetucker** [comment]
Actually, looking at this a bit more, it's simpler than I anticipated.  Django already handles the popup/iframe cases, so all that we need to do is override the template admin/inc/branding.html that is provided by djangocms-admin-style to include all of the CMS toolbar links, not just "Change password" and "Logout" links.

**Comment by leetucker** [comment]
OK, actually, we will also still need to override admin/base_site.html to ensure that django-sekizai blocks are rendered in the appropriate place in admin.  However, I've confirmed that getting the CMS toolbar to show up is straightforward by overriding templates/admin/base_site.html and templates/admin/inc/branding.html.

I'm not sure why this isn't default behavior for the djangocms-admin-style package (I think it just may be that configurable toolbars are relatively new in Django CMS), so I've posted an issue to that repo to see if they have plans to just go ahead and implement this improvement within that package.  If they end up doing so, then I may not make any changes to this repo.  If they aren't going to change that repo anytime soon, then I'll figure out how best to implement this change in a way that doesn't complicate project setup too much.
